### DISCUSSION TITLE
v0.0.10

### DISCUSSION THREAD

**Discussion by EdoFro** [EdoFro/Freeplane_Map-Drive-Inator#29] [Announcements]
## v0.0.10
* Changed script condition for "file" style in '**MDI styles template.mm**' to also work with relative file links
* added MDI.version
* added log info to "new imports" node's note
* createFilesFromNotes: ask if extension text is correct
* createFilesFromNotes: request confirmation if file already exists
* MDI.statusInfo() also writes the message into Freeplane's Log
* bug correction: added method getPathFromLink4 that solves issues when file.getCanonicalPath fails

<hr /><em>This discussion was created from the release <a href='https://github.com/EdoFro/Freeplane_Map-Drive-Inator/releases/tag/v0.0.10'>v0.0.10</a>.</em>

**Comment by bepolymathe** [comment]
Hi @EdoFro 

After a quick test on linux I have the impression that the new version poses again an old problem that you had fixed. If you put a task (GTD plugin) in a folder it is always recognized as a broken link (like "broken/missing"). 

Can you reproduce this?

**Reply by EdoFro** [reply]
this [issue](https://github.com/EdoFro/Freeplane_Map-Drive-Inator/issues/15#issue-1564591922)?

**Comment by bepolymathe** [comment]
Yes, that's the one. If I create a node (e.g. a task) without an MDI link, it's not a problem. But if I add a link (to an email for example) MDI detects the node as a file... and so gives it the style of a broken link.

**Reply by bepolymathe** [reply]
This is the case with a link to an ID or link to mail but not with a link to a web page for example.

**Reply by EdoFro** [reply]
can you reimport the MDI styles and tell me if that helped?

**Reply by bepolymathe** [reply]
Okay, I went deeper into the tests and I think I got it. Re-importing the styles didn't change anything so I did some tests with different types of links. The only link that kept causing problems was the one with a truncated ID that doesn't match a freeplane ID or link, nor a link recognized by the OS. In short it was my copy/paste that was broken and the link is not clean so logically interpreted by MDI as broken. So it works. Sorry for the inconvenience.

**Reply by EdoFro** [reply]
No problem! Nice to know it works!
